### PR TITLE
Add weld@0.2.3 w/ git auto-update

### LINKED ISSUE #10607: [Request] Add weld
**Library name:** weld
**Git repository url:** https://github.com/tmpvar/weld
**npm package url(optional):** https://www.npmjs.com/package/weld
**License(s):** MIT, [ref](https://github.com/tmpvar/weld#licence)
**Wanna say something? Leave message here:**
cc #10165 
@nolsherry please help me review this issue, thanks!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #10607, cc @tmpvar

Pull request for issue: #10607

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/tmpvar/weld
 * NPM package url (optional): https://www.npmjs.com/package/weld
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 11
   - Count of stars: 690
   - Count of forks: 37
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 17
   - Downloads in the last month: 141

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by dakshshah96** [comment]
@clairetsai818 Please check again, the latest version is 0.2.3 on the git repository.

@sufuf3 Done 👍

File: ajax/libs/weld/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "author": "hij1nx <hij1nx@gmail.com>",
+  "license": "MIT",
+  "filename": "weld.min.js",
+  "keywords": [
+    "templates",
+    "templating",
+    "jsdom",
+    "unobtrusive"
+  ],
```
Comment (hare1039) [inline ajax/libs/weld/package.json]: I think we can add `https://www.weld.io/` as `homepage`

Comment (dakshshah96) [inline ajax/libs/weld/package.json]: @hare1039 I don't think `https://www.weld.io/` is the official homepage for this project. Are you sure?

File: ajax/libs/weld/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "name": "weld",
+  "version": "0.2.3",
+  "description": "Template antimatter for JavaScript",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tmpvar/weld.git"
+  },
+  "author": "hij1nx <hij1nx@gmail.com>",
+  "license": "MIT",
+  "filename": "weld.min.js",
```
Comment (extend1994) [inline ajax/libs/weld/package.json]: Could you move this line to the second line? It's simpler to see and be revised if needed.

File: ajax/libs/weld/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "unobtrusive"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/tmpvar/weld.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "weld?(.min).js"
```
Comment (extend1994) [inline ajax/libs/weld/package.json]: Please use `weld*` because it is more flexible. Thanks.

**Comment by dakshshah96** [comment]
@extend1994 Done 👍

File: ajax/libs/weld/package.json (hunk 1 above)
Comment (hare1039) [inline ajax/libs/weld/package.json]: OK

**Comment by dakshshah96** [comment]
@pvnr0082t Done 👍 

* Fixed file to match upstream.

**Comment by dakshshah96** [comment]
Is this good to merge?

**Comment by PeterDaveHello** [comment]
@pvnr0082t I think two of @cdnjs/intern3 reviews could be fine.

**Comment by PeterDaveHello** [comment]
LGTM

**Pull request merged**